app/src/main/java/com/example/caregiver/Signup.java

```diff
@@ -117,6 +136,12 @@ public void callFirebase(String email, String password, String name){
                 userObject.put(user.getUid(), new User(name, email, userRole));
                 usersRef.updateChildren(userObject);
 
+                if (UUIDValue != null)
+                {
+                    Log.i("UUID", UUIDValue);
```

Comment:
Just remove the log but otherwise this is a great change! Thanks for adding this in Jui!